app/controllers/organizations_controller.rb

```diff
@@ -23,8 +23,33 @@ def create
       ).to_s
       redirect_to url_with_subdomain
     else
+      flash[:alert] = @organization.errors.full_messages.join(', ')
       render :new
-      flash[:alert] = t('organization.create.failure_message')
+    end
+  end
+
+  def login
+    # This action renders the form where the user inputs their organization site
+  end
+
+  def process_login
```

Comment:
Whats the difference between login and process_login, action name should be comprehensive.